### ISSUE TITLE
startup fails altogether

### ISSUE THREAD

**Issue by mikeglenn** [ChrisAnd1998/FalconX-Center-Taskbar#106]
I saw another issue posted about slow startup and opening with a visible startup screen, but in my issue is FalconX intermittently doesn't start at all after reboot. Even though FalconX appears in the Task Manager startup tab, I sometimes have to start it manually. This started happening sometime within the last month or two.

**Comment by ChrisAnd1998** [comment]
Well there have been some issues with version 1.3.3.0.
So the only thing i can think of is that you're still using 1.3.3.0. 

Can you give me more info about your current Windows 10 installation if this is not the case?
Or is it getting blocked by any AV?

**Comment by sakt1moko** [comment]
Same problem here. Windows 10 pro, updated. 
Ive got the windows store version

**Comment by ChrisAnd1998** [comment]
@sakt1moko yep, store version is still on 1.3.3.0.
You have to wait for Microsoft to finally publish my submission.

**Comment by mikeglenn** [comment]
Yes, sorry for the lack of details. You're correct about the Falconx version. I'm running Windows Defender and Malwarebytes Pro, but I don't see anything in the logs about FalconX.

- FalconX: 01.12.2019 - 1.3.3.0.
- Windows 10: 1909 build 1836.476

**Comment by ChrisAnd1998** [comment]
Update has been published.
